sdk/core/azure-core/inc/http/curl/curl.hpp

```diff
@@ -14,7 +14,8 @@
 
 namespace Azure { namespace Core { namespace Http {
 
-  constexpr auto UploadStreamPageSize = 1024;
+  // libcurl CURL_MAX_WRITE_SIZE is 16k.
```

Comment:
This says 16K and the line below is 64K

Reply:
type. should say 64k on the comment. thank you